src/modules/fw_autotune_attitude_control/fw_autotune_attitude_control.hpp

```diff
@@ -177,6 +192,9 @@ class FwAutotuneAttitudeControl : public ModuleBase<FwAutotuneAttitudeControl>,
 	float _filter_sample_rate{1.f};
 	bool _are_filters_initialized{false};
 
+
+	float signal{0.0f};
```

Comment:
does it really need to be a member of the class, you're not using that variable anywhere else than in `getIdentificationSignal()` right?

Reply:
No we dont new this as class member, we can use at just getIdentificationSignal() function